bundle/src/main/java/com/adobe/acs/commons/wcm/impl/SiteMapServlet.java

```diff
@@ -88,6 +87,8 @@
 
     private static final boolean DEFAULT_REMOVE_TRAILING_SLASH = false;
 
+    private static final boolean DEFAULT_DONT_USE_VANITY_URL = false;
```

Comment:
double negation ... I would prefer a positive statement like
```
DEFAULT_DO_USE_VANITY_URL = true;
```
and use the same positive logic below as well. Much easier to understand.